packages/component-meta/tests/unit/specs/__snapshots__/componentMeta.spec.ts.snap

```diff
@@ -2092,14 +2093,14 @@ exports[`should match snapshot 1`] = `
     "isInternal": false,
     "isThemeable": true,
     "namedSlots": [
+      "hidden-select",
```

Comment:
This isn't user facing I think. 
So it should be removed here or even better be moved into `internalNamedSlots`?